crates/iota-adapter-transactional-tests/tests/shared/by_value_shared_object_deletion_via_merge_fails.move

```diff
@@ -51,7 +51,7 @@ module t2::o2 {
         df::add(&mut parent.id, 0, o2);
     }
 
-    public fun transferer(o2: Coin<IOTA>) {
+    public fun transferrer(o2: Coin<IOTA>) {
```

Comment:
This should be just `transfer`

Reply:
true, good call

Reply:
although if you look at the naming scheme they are all like this

Reply:
Hm true, but then it should stay as before

Reply:
no, I don't think so

Reply:
To me it looks like they just added an `r` at the end for some methods, but since it's wrong anyways I'll resolve this comment